Demo/Demo/Fullscreen/SearchResults/SearchResultsData.swift

```diff
@@ -0,0 +1,15 @@
+import FinnUI
+
+class RecentSearchData: SearchResultsListViewModel {
+    let title = "Siste søk"
+    let icon = UIImage(named: .arrowCounterClockwise)
```

Comment:
If the icon is used by the Demo project it should be a resource in there, the long term plan is to be able to remove the "public" modifier from `FinniversImageAsset`, now we have to little control over which image is used where

Reply:
Yes, it is used in the demo, but also in the ios-app. I will move this icon to the demo then, and also add the icon to ios-app.